internal/command/logs/logs.go

```diff
@@ -50,9 +51,9 @@ Use --no-tail to only fetch the logs in the buffer.
 		flag.Region(),
 		flag.JSONOutput(),
 		flag.String{
-			Name:        "instance",
-			Shorthand:   "i",
-			Description: "Filter by instance ID",
+			Name:        "machine",
+			Description: "Filter by machine ID",
+			Aliases:     []string{"instance"},
```

Comment:
We should also avoid breaking `-i`.

Reply:
Looking at this, you'll probably have to make some changes to [this code](https://github.com/superfly/flyctl/blob/master/internal/flag/flag.go#L16-L35) to make `-i` work, since one letter `-` flags need to be specified in the `Shorthand` struct field and not the `Name` field.

Reply:
Oh! Thanks for the eagle eyes @btoews ! I didn't consider the shorthand for instance

Reply:
Added a new flag.String attribute for this! 

> Add a boolean attribute [UseAliasShortHand](https://github.com/superfly/flyctl/pull/3820/files#diff-cd4ae47be757809e391f0b84469d6a2e9a6299fb56bacc30164a89119ff5fb4eR110) in flag.String struct. And [read this in makeAlias()](https://github.com/superfly/flyctl/pull/3820/files#diff-cd4ae47be757809e391f0b84469d6a2e9a6299fb56bacc30164a89119ff5fb4eR40) to determine whether to include aliases' first characters as shorthand